.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Discord Community
```

Comment:
Not sure if needed, this was added when I was trying to make a discord server, but didn't go any well 😓 So Maybe remove this

Reply:
Coool Cool!!